src/Test.FunctionalTests.Sarif/PropertyBagConverterTests.cs

```diff
@@ -21,9 +22,17 @@ public void PropertyBagConverter_RoundTripsStringPropertyWithEscapedCharacters()
             string stringPropertyName = nameof(stringPropertyName);
             string stringPropertyValue = "'\"\\'";
 
+            string normalStringPropertyName = "source.language";
+            string normalStringPropertyValue = "csharp";
+
+            string dateTimePropertyName = "runStart";
+            DateTime dateTimePropertyValue = DateTime.UtcNow.AddHours(-1.43);
+
             var run = new Run();
             run.SetProperty(intPropertyName, 42);
             run.SetProperty(stringPropertyName, stringPropertyValue);
+            run.SetProperty(normalStringPropertyName, normalStringPropertyValue);
+            run.SetProperty<DateTime>(dateTimePropertyName, dateTimePropertyValue);
```

Comment:
>DateTime [](start = 28, length = 8)

The compiler will infer the generic type from the type of the second parameter, but I understand if you want to be explicit in the context of test code (to demonstrate the types you're covering).